pkg/virt-handler/vm.go

```diff
@@ -559,6 +559,7 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 			for _, existingInterfaceSpec := range vmi.Spec.Domain.Devices.Interfaces {
 				existingInterfacesSpecByName[existingInterfaceSpec.Name] = existingInterfaceSpec
 			}
+
```

Comment:
nit: not related change.

Reply:
Thanks, just thought that since we use those two, it worth to space between them in this PR, up to you

Reply:
It is not close enough to the code that was changed in this PR so it is hard to ignore it.

Reply:
right, i meant close to the code logic, np

Reply:
It got back?

Reply:
nope sorry forgot to remove